src/System.CommandLine.DragonFruit/CommandLine.cs

```diff
@@ -116,6 +116,18 @@ public static class CommandLine
                 command.AddOption(option);
             }
 
+            if (method.GetParameters()
```

Comment:
This ends up with the parameter being included as both an argument and an option. Is that what was intended? I think at a minimum we should have another test around this behavior (either including these parameters as options or not).

Reply:
Ah, good point. I'll fix that.